cli/slsa-verifier/verify.go

```diff
@@ -32,7 +32,7 @@ func verifyArtifactCmd() *cobra.Command {
 	o := &verify.VerifyOptions{}
 
 	cmd := &cobra.Command{
-		Use: "verify-artifact",
+		Use: "verify-artifact [flags] artifact",
```

Comment:
Do `[flags]` have to come after `artifact`, as `verify-artifact artifact [flags]`. That's what the doc says https://github.com/slsa-framework/slsa-verifier#artifacts, but maybe what you wrote works too?

Reply:
I usually put them before, they can come after too. In general, it seems Linux prefers to put the options (`[flags]`) before the arguments in help messages, even if they could be used after:

```
[~] λ ls --help
Usage: ls [OPTION]... [FILE]...
...
```

```
[~] λ gcc --help
Usage: gcc [options] file...
Options:
...
```

So I thought I'd follow the same approach

Reply:
/cc @asraa

Reply:
with Cobra it can be either or! I prefer putting arguments before as well +1